src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -235,166 +253,127 @@ public Trigger intakeHasUnalignedCoralTrigger() {
+              setConveyerMotor(0);
+              setIntakeMotor(0);
+            })
+        .withName("conveyerOutCommand");
   }
 
   public Command floorIntake() {
     return startEnd(
-        () -> {
-          goToPivotPosition(kPivotExtendAngle);
-          intakeMotor.set(kIntakeSpeed);
-        },
-        () -> {});
+            () -> {
+              goToPivotPosition(kPivotExtendAngle);
+              setIntakeMotor(kIntakeSpeed);
+              if (Robot.isSimulation()) intakeSim.startIntake();
+            },
+            () -> {
+              goToPivotPosition(kPivotRetractAngle);
+            })
+        .withName("floorIntake");
   }
 
   public Command floorOuttake() {
     return startEnd(() -> goToPivotPosition(kPivotExtendAngle), () -> {})
         .until(pivotAtSetpoint(kPivotExtendAngle))
-        .andThen(() -> intakeMotor.set(kOuttakeSpeed));
+        .andThen(() -> setIntakeMotor(kOuttakeSpeed))
+        .withName("floorOuttake");
   }
 
   public Command humanPlayerIntake() {
     return startEnd(
-        () -> {
-          goToPivotPosition(kcoralStation);
-          intakeMotor.set(kIntakeSpeed);
-        },
-        () -> {});
+            () -> {
+              goToPivotPosition(kcoralStation);
+              setIntakeMotor(kIntakeSpeed);
+            },
+            () -> {})
+        .withName("humanPlayerIntake");
   }
 
   public Command algaeIntake() {
     return startEnd(
-        () -> {
-          goToPivotPosition(kalgae);
-          intakeMotor.set(-kIntakeSpeed);
-        },
-        () -> {});
+            () -> {
+              goToPivotPosition(kAlgae);
+              setIntakeMotor(-kIntakeSpeed);
+            },
+            () -> {})
+        .withName("algaeIntake");
   }
 
   public Command algaeHold() {
     return startEnd(
-        () -> {
-          goToPivotPosition(kalgae);
-          intakeMotor.setControl(new TorqueCurrentFOC(kHoldPower));
-        },
-        () -> {});
+            () -> {
+              goToPivotPosition(kAlgae);
+              intakeMotor.setControl(new TorqueCurrentFOC(kHoldPower));
+            },
+            () -> {})
+        .withName("algeaHold");
   }
 
   public Command algaeOuttake() {
     return startEnd(
-        () -> {
-          goToPivotPosition(kalgae);
-          intakeMotor.set(kIntakeSpeed);
-        },
-        () -> {});
+            () -> {
+              goToPivotPosition(kAlgae);
+              setIntakeMotor(kIntakeSpeed);
+            },
+            () -> {})
+        .withName("algaeOuttake");
   }
 
   public Command l1ScoreModeA() {
-    return startEnd(
-        () -> {
-          goToPivotPosition(kl1);
-          intakeMotor.set(-kIntakeSpeed);
-        },
-        () -> {});
+    return startEnd(() -> goToPivotPosition(kPivotL1Score), () -> {})
+        .until(pivotAtSetpoint(kPivotL1Score))
+        .andThen(() -> setIntakeMotor(kOuttakeSpeed))
+        .finallyDo(() -> goToPivotPosition(kPivotRetractAngle))
```

Comment:
Let the Idle Command pull the pivot arm back, don't need the finallyDo. 
The setIntakeMotor should be in a startEnd Command so it doesn't immediately end